watchbuild/lib/watchbuild/options.rb

```diff
@@ -8,6 +8,10 @@ def self.available_options
       user ||= CredentialsManager::AppfileConfig.try_fetch_value(:apple_id)
 
       [
+        FastlaneCore::ConfigItem.new(key: :sample_only_once,
+                                     description: "Will sample iTunes Connect only once and won't loop every 30 seconds",
```

Comment:
Let's change that to: `Only check for the build once, instead of waiting for it to process.`